install_script.sh.template

```diff
@@ -549,9 +588,16 @@ function install_installer() {
     (_install_installer "$sudo_cmd" "$os" "$distribution" > /tmp/datadog-installer-stdout.log 2> /tmp/datadog-installer-stderr.log) || exit_status=$?
     exit_status=${exit_status:-0}
 
+    local packages_to_install
+    local pacakges_to_install_after_installer
```

Comment:
same typo, but this one might actually be problematic since you refer to the correct variable name afterward